html-forms.html

```diff
@@ -0,0 +1,152 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>HTML Forms Assignment</title>
+
+  <style>
+    .form {
+      width: 33%;
+      margin: 0 auto;
+    }
+
+    fieldset {
+      border-color: #0cb5c9;
+      border-radius: 10px;
+      padding: 25px
+    }
+
+    .form-field legend {
+      color: #32757d;
+      font-style: italic;
+      font-weight: 600;
+    }
+
+    label {
+      font-weight: 600;
+    }
+
+    input,
+    select {
+      margin: 10px 0;
+      border-radius: 5px;
+      border: 1px solid #ccc;
+    }
+
+    input[type="password" i],
```

Comment:
Flag "i" sets the selector to be case insensitive.  This way you let yourself write input types in any case which is bad practice. So better not to use this flag in order to have a warning about the wrong input type case